src/tests/test_ffi.cpp

```diff
@@ -247,6 +248,40 @@ class FFI_Unit_Tests final : public Test
             TEST_FFI_OK(botan_rng_add_entropy, (rng, not_really_entropy, 32));
             }
 
+         size_t cb_counter = 0;
+
+         auto custom_get_cb = +[](uint8_t* out, size_t out_len, void* context) -> int
```

Comment:
TIL about `+[]` to force the lambda to be a function pointer, hadn't seen that before.

Reply:
TBH: Found on SO